### ISSUE TITLE
Ubuntu > 16 on github actions not possible

### ISSUE THREAD

**Issue by luca-rath** [handcraftedinthealps/ZendSearch#5]
Upgrading Ubuntu for github actions to something higher than 16.04 is not possible, because the iconv implementation changed, which leads to failing tests. For now we will stick with Ubuntu 16.04. Surprisingly this only happens on PHP 7, not on PHP 8

PR: https://github.com/handcraftedinthealps/ZendSearch/pull/4

Output:

```
2021-01-07T13:22:52.3690207Z PHPUnit 5.7.27 by Sebastian Bergmann and contributors.
2021-01-07T13:22:52.3699735Z 
2021-01-07T13:22:52.3708132Z ...............................................................  63 / 157 ( 40%)
2021-01-07T13:22:55.2587687Z .........................EE.................................... 126 / 157 ( 80%)
2021-01-07T13:22:58.0331299Z ...............................                                 157 / 157 (100%)
2021-01-07T13:22:58.0331755Z 
2021-01-07T13:22:58.0332143Z Time: 6.32 seconds, Memory: 22.00MB
2021-01-07T13:22:58.0332655Z 
2021-01-07T13:22:58.0333004Z There were 2 errors:
2021-01-07T13:22:58.0333281Z 
2021-01-07T13:22:58.0334618Z 1) ZendSearchTest\Lucene\IndexTest::testAddDocument
2021-01-07T13:22:58.0339442Z iconv(): Detected an illegal character in input string
2021-01-07T13:22:58.0339888Z 
2021-01-07T13:22:58.0340592Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Analysis/Analyzer/Common/Text.php:42
2021-01-07T13:22:58.0341798Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Analysis/Analyzer/AbstractAnalyzer.php:69
2021-01-07T13:22:58.0343067Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index/SegmentWriter/DocumentWriter.php:84
2021-01-07T13:22:58.0344207Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index/Writer.php:231
2021-01-07T13:22:58.0345166Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index.php:1132
2021-01-07T13:22:58.0346119Z /home/runner/work/ZendSearch/ZendSearch/tests/ZendSearch/Lucene/IndexTest.php:291
2021-01-07T13:22:58.0346684Z 
2021-01-07T13:22:58.0347359Z 2) ZendSearchTest\Lucene\IndexTest::testOptimize
2021-01-07T13:22:58.0350909Z iconv(): Detected an illegal character in input string
2021-01-07T13:22:58.0351346Z 
2021-01-07T13:22:58.0352056Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Analysis/Analyzer/Common/Text.php:42
2021-01-07T13:22:58.0353231Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Analysis/Analyzer/AbstractAnalyzer.php:69
2021-01-07T13:22:58.0354510Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index/SegmentWriter/DocumentWriter.php:84
2021-01-07T13:22:58.0355644Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index/Writer.php:231
2021-01-07T13:22:58.0356486Z /home/runner/work/ZendSearch/ZendSearch/library/ZendSearch/Lucene/Index.php:1132
2021-01-07T13:22:58.0357303Z /home/runner/work/ZendSearch/ZendSearch/tests/ZendSearch/Lucene/IndexTest.php:335
2021-01-07T13:22:58.0359044Z 
2021-01-07T13:22:58.0359358Z ERRORS!
2021-01-07T13:22:58.0361128Z Tests: 157, Assertions: 1068, Errors: 2.
```

**Comment by YetiCGN** [comment]
What about 22.04 LTS now?

**Comment by luca-rath** [comment]
Thanks for reminding me of this, seems to work just fine 🙏 Will be fixed by #7
